runtime/src/qio/qio_formatted.c

```diff
@@ -4724,6 +4726,8 @@ qioerr qio_conv_parse(c_string fmt,
       if( specifier == 't' ) {
         // Does nothing at all with width.
         spec_out->argType = QIO_CONV_ARG_TYPE_REPR;
+      } else if ( specifier == '?' ) {
```

Comment:
If I understand the backing issue correctly we should prohibit (not just ignore) width / precision specifiers. Does this have that behavior? Additionally, I don't think I saw a test that ensures that; if there isn't one, I think you should add it.

Reply:
Good point, I'll see if I can add that.